Snowflake.Data.Tests/UnitTests/SFDbCommandTest.cs

```diff
@@ -34,6 +34,19 @@ public void TestCommandWithConnectionAndCommandText()
             Assert.AreEqual(commandText, command.CommandText);
         }
 
+        [Test]
+        public void TestCommandExecuteThrowsExceptionWhenCommandTextIsNotSet()
```

Comment:
please add also a test for async command execution

Reply:
Added